2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,15 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(oxLevels) {
+   return oxLevels.find((level) => {
+     return (
+       Number(level.replace("%", "")) > 19.5 &&
+       Number(level.replace("%", "")) < 23.5 &&
+       level.includes("%")
+     );
+   });
+}
 
```

Comment:
This is good clean looking code. I made a couple of small changes which I'll explain below.
```javascript
function findSafeOxygenLevel(oxLevels) {
  const lowerLimit = 19.5;
  const upperLimit = 23.5;
  return oxLevels.find((level) => {
    return (
      level.includes("%") &&
      Number(level.replace("%", "")) > lowerLimit &&
      Number(level.replace("%", "")) < upperLimit
    );
  });
}
```
In large functions numeric literals like "19.5" can be "hidden" in the code. I've created a variable representing these values to make the function more readable.
The predicate function used by ```oxLevels.find``` checks three conditions. When Javascript evaluates expressions that are &&'d together, it evaluates them left-to-right. I've moved the percentage check to be first so that if that expression returns false, the other two expressions are not evaluated. This technique speeds up your function and is sometimes known as "short circuiting" an expression. You could also have used ```.endsWith("%")``` instead of includes.